packages/aws-cdk-lib/aws-ec2/lib/launch-template.ts

```diff
@@ -146,6 +146,53 @@ export enum SpotRequestType {
   PERSISTENT = 'persistent',
 }
 
+/**
+ * Capacity Reservation preferences
+ */
+export enum CapacityReservationPreference {
+  /** Launch instances into any available Capacity Reservation */
+  OPEN = 'open',
+  /** Do not launch instances into Capacity Reservations */
+  NONE = 'none',
+  /** Only launch instances into Capacity Reservations */
+  CAPACITY_RESERVATIONS_ONLY = 'capacity-reservations-only',
+}
+
+/**
+ * Target Capacity Reservation specification
+ */
+export interface LaunchTemplateCapacityReservationTarget {
+  /**
+   * The ID of the Capacity Reservation in which to run the instance
+   * @default - No specific Capacity Reservation ID
+   */
+  readonly capacityReservationId?: string;
+
+  /**
+   * The ARN of the Capacity Reservation resource group in which to run the instance
+   * @default - No Capacity Reservation resource group
+   */
+  readonly capacityReservationResourceGroupArn?: string;
+}
+
+/**
+ * Capacity Reservation specification for LaunchTemplate
+ * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-launchtemplate-launchtemplatedata-capacityreservationspecification.html
+ */
+export interface LaunchTemplateCapacityReservationSpecification {
+  /**
+   * Indicates the instance's Capacity Reservation preferences
+   * @default - No preference specified
```

Comment:
Surely there is an implied default here. Tell us what the default is.

> Describe the default value or default behavior, even if it's not CDK that controls the default. For example, if an absent value does not get rendered into the template and it's ultimately the AWS service that determines the default behavior, we still describe it in our documentation.

https://github.com/aws/aws-cdk/blob/main/docs/DESIGN_GUIDELINES.md#defaults

"No XXX specified" is not useful to say. Here's what you're telling a user:

"If you don't specify a capacityReservationPreference, then no capacityReservationPreference is specified."

I think we can trust our customers to figure that out for themselves. What they want to know is *what happens* when they don't specify a certain thing.